src/plugins/divclass.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * ### Divclass
+ * 
+ * We support the ability to wrap content in divs with a declared class
+ * attribute. Any kind of block content is supported; to wrap the content, simply
+ * precede it with a block entirely consisting of the class name wrapped in square
+ * bracked and succeed it with a block entirely consisting of the class name
+ * preceeded by a forward slash and wrapped in square brackets.
+ * 
+ * [col-33]
+ * 
+ * This is some simple content that will be wrapped in a div with the class "col-33"
+ * 
+ * [/col-33]
+ * 
+ * [col-33]
+ * 
+ * - This is a ul, which will also be wrapped in a div with the class "col-33".
+ * - Note that because this is dealing with classes and not ids, duplicate class
+ * - names are just fine.
+ * 
+ * [/col-33]
+ * 
+ * [outer]
+ * 
+ * [inner]
+ * 
+ * Nesting of classes is also supported.
+ * 
+ * [/inner]
+ * 
+ * [/outer]
+ * 
+ * [unsupported]
+ * 
+ * Some things that are NOT supported:
+ * 
+ * - inline divclasses like [example]this[/example]
+ * - generic endings like [example]\n\ncontent\n\n[/]
+ * - indentation like [example]\n\n    content\n\n[/example]
+ *   - in this case, 'content' would be treated as a code block
+ * 
+ * [/unsupported]
+ *
+ * Note that you can also add an empty div without content like so:
+ *
+ * [empty]
+ *
+ *
+ *
+ * [/empty]
+ *
+ * ### Redacting
+ *
+ * Divclasses will be redacted to just [] and [/]:
```

Comment:
I'm unsure if this comment is accurate in its context or not - in your tests, we seem to consistently include an index in the redaction, but if that concern is a layer above this plugin this makes sense.

Reply:
oops, right you are; this comment is outdated